packages/components/icons/src/icons.visualroute.jsx

```diff
@@ -83,7 +83,7 @@ const renderIcon = (iconName, color, size) => {
 };
 
 export const component = () => (
-  <Switch>
+  <Routes>
```

Comment:
This is a full switch over to the v6 API. This was a straightforward route that I didn't see anything else that needed to be migrated, but let me know if I missed anything. Hard to test as I think these are the Percy screenshot tests that I don't have access to run?